github/actions_oidc.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// OIDCSubjectClaimCustomizationTemplate represents an OIDC subject claim customization template
+type OIDCSubjectClaimCustomizationTemplate struct {
+	UseDefault       *bool    `json:"use_default,omitempty"`
+	IncludeClaimKeys []string `json:"include_claim_keys"`
+}
+
+// GetOrganizationOIDCSubjectClaimCustomizationTemplate gets the subject claim customization template for an organization.
+//
+// GitHub API docs: https://docs.github.com/en/rest/actions/oidc#get-the-customization-template-for-an-oidc-subject-claim-for-an-organization
+func (s *ActionsService) GetOrganizationOIDCSubjectClaimCustomizationTemplate(ctx context.Context, org string) (*OIDCSubjectClaimCustomizationTemplate, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/actions/oidc/customization/sub", org)
+	return s.getOIDCSubjectClaimCustomizationTemplate(ctx, u)
+}
+
+// GetRepositoryOIDCSubjectClaimCustomizationTemplate gets the subject claim customization template for a repository.
```

Comment:
Would it bother you if we shortened this to `GetRepoOIDCSubjectClaimCustomTemplate` ?